src/parser-json-to-ast.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+function parse(text) {
+  const jsonToAst = require("json-to-ast");
+  const ast = jsonToAst(text);
+  toBabylon(ast);
```

Comment:
Can you avoid doing the mutations and instead recreate those objects using Object.assign. It's way easier to debug when we don't mutate things.